src/os/bluestore/BlueStore.cc

```diff
@@ -7815,6 +7816,10 @@ void BlueStore::_txc_finish(TransContext *txc)
 	  // for _osr_drain_preceding()
           notify = true;
 	}
+	if (txc->state == TransContext::STATE_DEFERRED_QUEUED &&
```

Comment:
what's about an alternative approach to submit deferred txcs if deferred queue hasn't been updated for some time?